examples/platform_integration/platform_views/lib/native_view_example_4.dart

```diff
@@ -0,0 +1,53 @@
+// ignore_for_file: unused_local_variable
+
+// #docregion import
+import 'package:flutter/foundation.dart';
```

Comment:
Is this part of a project? We use flutter projects to confirm the code is up to date as flutter changes.

Reply:
I couldn't find any related projects for this, but can confirm the code passes analysis and compiles, similar to the iOS code in `native_view_example_3.dart`. I couldn't find any project on GitHub that included this code though.

Reply:
We should wrap it in a project here. @khanhnwin can give progress on how to do this in a way that makes sure this code will stay correct going forward.

Reply:
@khanhnwin currently the Dart code is checked into the examples directory similar to the iOS/Android versions, but the Swift code is not (also similar to the iOS/Android versions). Where's the best place to land that code?

@domesticmouse do you think we should hold off landing this until that happens, or follow up with separate PRs that lands macOS, iOS, and Android versions of the native code as well?

Agreed we should do this though -- writing the Swift code for this involved basically recreating the exact project you're asking for, just to be sure the code is actually correct.